Makefile

```diff
@@ -15,6 +15,7 @@ OPERATOR_SDK_VERSION=0.12.0
 AUTH_TOKEN=$(shell curl -sH "Content-Type: application/json" -XPOST https://quay.io/cnr/api/v1/users/login -d '{"user": {"username": "$(QUAY_USERNAME)", "password": "${QUAY_PASSWORD}"}}' | jq -r '.token')
 TEMPLATE_PATH="$(shell pwd)/templates/monitoring"
 INTEGREATLY_OPERATOR_IMAGE ?= $(REG)/$(ORG)/$(PROJECT):v$(TAG)
+KUBECONFIG="${HOME}/.kube/config"
```

Comment:
I think the prow failures might be related to this, although I'm not 100% sure on that. It's like it's targeting the wrong cluster now.

Reply:
Ok, good idea, I'll try to remove it from this Makefile and run the command inside the `ocm/install/rhmi-addon` target